core/src/main/java/com/yahoo/oak/Chunk.java

```diff
@@ -167,10 +181,10 @@ boolean allocateEntry(ThreadContext ctx, K key) {
     }
 
     /**
-     * See {@code EntrySet.writeValueStart(ThreadContext)} for more information
+     * See {@code EntrySet.allocateValue(ThreadContext)} for more information
      */
     void writeValue(ThreadContext ctx, V value, boolean writeForMove) {
```

Comment:
Probably should rename to writeValueStart...

Reply:
I'm indifferent to the name, but I suggest matching the names between `Chunk.writeValue` to `EntrySet.allocateValue`.
If you rename it to `Chunk.writeValueStart`, then also rename `EntrySet.writeValueStart`.

Reply:
all changed to `allocateValue`